src/components/DpUploadFiles/DpUploadFiles.vue

```diff
@@ -227,6 +228,9 @@ export default {
   data () {
     return {
       fileHashes: [],
+      TransKeys: {
```

Comment:
Here and above

```suggestion
      translations: {
```

Reply:
I did that at first, but after that I got error:
```
[Vue warn]: The data property "translations" is already declared as a prop. Use prop default value instead.

found in

---> <DpUploadFiles> at src/components/DpUploadFiles/DpUploadFiles.vue
```

Reply:
Done in https://github.com/demos-europe/demosplan-ui/commit/433f62d244703aa6cb9edf9a038078bffa739140